config/config.yaml

```diff
@@ -1,5 +1,6 @@
 project_code: sage-bionetworks
 profile: {{ var.profile | default("default") }}
 region: {{ var.region | default("us-east-1") }}
+template_bucket_name: bootstrap-awss3cloudformationbucket-zov6lb1ge7ll
```

Comment:
i don't think you want to do this because there are two AWS accounts for bridge.  There's an account for dev & uat and a 2nd account for prod env.  Those accounts have different cloudformation buckets that's why this is defined in multiple `config.yaml` files